docs/rst/manual/about/release_notes.rst

```diff
@@ -27,6 +27,106 @@ Release Notes
  Miscellaneous:
    -
 
+Version 2.34
+------------
+
+Breaking changes:
+  - Remove the IDE (`#3148 <https://github.com/equinor/ert/pull/3148>`_)
```

Comment:
Maybe explain what an 'IDE' is, since this is a breaking change, and few users will understand from this that the editor is actually gone..

Reply:
I agree, I was planning to change the wording when I adapt the change notes for the komodo release, but I will amend the description a bit here as well :slightly_smiling_face: